lib/app.rb

```diff
@@ -153,4 +150,20 @@ def run
       command = @input_stream.gets.chomp
     end
   end
+
+  private
+
+  def switch_current_project
+    output_stream.puts("\e[0;35mEnter a project name:\e[0m")
```

Comment:
I wanted to make a `prompt` method, but the different colors used throughout seemed like too much of a barrier to cross for now.